docs/tutorials/java/mxnet_java_on_intellij.md

```diff
@@ -0,0 +1,210 @@
+# Run MXNet Java Examples Using the IntelliJ IDE (macOS)
+
+This tutorial guides you through setting up a simple Java project in IntelliJ IDE on macOS and demonstrates usage of the MXNet Java APIs. 
+
+## Prerequisites:
+To use this tutorial you need the following pre-requisites:
+
+- [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Maven](https://maven.apache.org/install.html)
+- [OpenCV](https://opencv.org/)
+- [IntelliJ IDE](https://www.jetbrains.com/idea/)
```

Comment:
can you also point to the community edition?

Reply:
Sure. Will do.

Reply:
Done.